Removal of velocity dependency leads to exceptions based on the lack of commons-lang

Removal of velocity dependency (#132 ) leads to exceptions based on the lack of commons-lang (witch at previous version was a dependency of velocity).

Caused by: java.lang.NoClassDefFoundError: org/apache/commons/lang/StringUtils
at com.jgeppert.struts2.jquery.components.Head.evaluateParams(Head.java:194)

I believe that commons-lang should be added as a direct dependency for struts2-jquery. And some day update to lang3 or just use java standards.

**As @lukaszlenart stated, Struts2 is using lang3, so code should be updated to use lang3 instead of lang.**

--- a/struts2-jquery-plugin/src/main/java/com/jgeppert/struts2/jquery/components/Head.java
+++ b/struts2-jquery-plugin/src/main/java/com/jgeppert/struts2/jquery/components/Head.java
@@ -21,7 +21,7 @@ package com.jgeppert.struts2.jquery.components;
 
 import com.opensymphony.xwork2.inject.Inject;
 import com.opensymphony.xwork2.util.ValueStack;
-import org.apache.commons.lang.StringUtils;
+import org.apache.commons.lang3.StringUtils;
 import org.apache.struts2.StrutsConstants;
 import org.apache.struts2.views.annotations.StrutsTag;
 import org.apache.struts2.views.annotations.StrutsTagAttribute;

--- a/struts2-jquery-richtext-plugin/src/main/java/com/jgeppert/struts2/jquery/richtext/actions/CkeditorUpload.java
+++ b/struts2-jquery-richtext-plugin/src/main/java/com/jgeppert/struts2/jquery/richtext/actions/CkeditorUpload.java
@@ -3,7 +3,7 @@ package com.jgeppert.struts2.jquery.richtext.actions;
 import com.opensymphony.xwork2.Action;
 import com.opensymphony.xwork2.ActionSupport;
 import com.opensymphony.xwork2.inject.Inject;
-import org.apache.commons.lang.ArrayUtils;
+import org.apache.commons.lang3.ArrayUtils;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.apache.struts2.ServletActionContext;

--- a/struts2-jquery-showcase/src/main/java/com/jgeppert/struts2/jquery/showcase/Autocompleter.java
+++ b/struts2-jquery-showcase/src/main/java/com/jgeppert/struts2/jquery/showcase/Autocompleter.java
@@ -22,7 +22,7 @@ package com.jgeppert.struts2.jquery.showcase;
 import com.jgeppert.struts2.jquery.showcase.model.Customer;
 import com.jgeppert.struts2.jquery.showcase.model.CustomerDAO;
 import com.opensymphony.xwork2.ActionSupport;
-import org.apache.commons.lang.StringUtils;
+import org.apache.commons.lang3.StringUtils;
 import org.apache.struts2.convention.annotation.Action;
 import org.apache.struts2.convention.annotation.Actions;
 import org.apache.struts2.convention.annotation.Result;

